sale_product_approval/models/sale_order.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2021 Open Source Integrators
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import date
+
+from odoo import SUPERUSER_ID, _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    exceptions_sale_approval_confirm = fields.Boolean(
+        compute="_compute_exceptions", string="Exception", default=False
+    )
+    override_exception = fields.Boolean(default=False)
```

Comment:
IMHO, this name is a little bit too broad. Should reflect the 'sale product approval' flow to avoid names collisions.

Reply:
No need of `default=False`as default